packages/core/util/tracks.ts

```diff
@@ -266,3 +272,92 @@ export function getTrackName(
   }
   return trackName
 }
+
+export function showTrackGeneric(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
this showTrackGeneric can basically be used for any view type, so is applied in dotplot, synteny, LGV, circular.

Reply:
maybe this could be an IAnyModelType? would at least avoid no-explicit-any

Reply:
added some better types and made a "GenericView" type here that says "view.tracks is an MSTArray of minimal track configs e.g. objects with a configuration.trackId"